srtp/crypto/openssl/hmac.cpp

```diff
@@ -75,7 +94,11 @@ void* initializeSha1HmacContext(void* ctx, uint8_t* key, int32_t keyLength)
 {
```

Comment:
`initializeSha1HmacContext(void* ctx,` ctx is actually a double-pointer. That's the address of a structure element which is of type `HMAC_CTX*`. I'd have to change the function parameters for that. 

I think the same goes for [`hmacSha1Ctx(void* ctx, const uint8_t* data[], uint32_t data_length[],
                 uint8_t* mac, int32_t* mac_length )		                 uint8_t* mac, int32_t* mac_length )`](https://github.com/wernerd/ZRTPCPP/pull/43/files#diff-f1f78da756612f02f65c069116f1ba2eR116)